runtime/onert/backend/train/ops/ElementwiseActivationLayer.cc

```diff
@@ -62,7 +64,16 @@ void ElementwiseActivationLayer::forward(bool) { cpu::ops::ElementwiseActivation
 
 void ElementwiseActivationLayer::backward(uint32_t)
 {
-  // TODO Implement details
+  switch (_op_type)
+  {
+    case ElementwiseActivationType::kReLU:
```

Comment:
Shouldn't it be implemented differently for each type?

Reply:
I didn't consider data type.

Reply:
I moved setting kernels to `configure` as cpu backend.